content/release-10-14-0/streaming-analytics-10-14-0-bundle/10_14_0.md

```diff
@@ -28,3 +28,10 @@ The [Measurement Output](https://documentation.softwareag.com/apama/Analytics_Bu
 now has a new **Time** input port. This can be used to set the timestamp of the measurement and works similar to the already existing **Time** input ports of the 
 [Alarm Output](https://documentation.softwareag.com/apama/Analytics_Builder/pab10-14-0/apama-pab-webhelp/index.html#page/apamaanalyticsbuilder-webhelp%2Fre_AnaBui_block_reference_Output_CreateAlarm.html) 
 and [Event Output](https://documentation.softwareag.com/apama/Analytics_Builder/pab10-14-0/apama-pab-webhelp/index.html#page/apamaanalyticsbuilder-webhelp%2Fre_AnaBui_block_reference_Output_CreateEvent.html) blocks.
+
+The [Measurement Input](https://documentation.softwareag.com/apama/Analytics_Builder/pab10-14-0/apama-pab-webhelp/index.html#page/apamaanalyticsbuilder-webhelp%2Fre_AnaBui_block_reference_Input_DeviceMeasurementInput.html) block 
+now also sends a new property named `value` as part of its output value. This allows other blocks, such as the 
+[Text Substitution](https://documentation.softwareag.com/apama/Analytics_Builder/pab10-14-0/apama-pab-webhelp/index.html#page/apamaanalyticsbuilder-webhelp%2Fre_AnaBui_block_reference_Utilities_TextSubstitution.html) block, 
+to use the measurement value (for example, to use it in an alarm text).
```

Comment:
2 x "to use"; I would remove the one in the brackets

Reply:
Let's keep it as it is. Otherwise I'll also have to change the English and German DITA sources. :-)